### DISCUSSION TITLE
[Solved] ."body could not be parsed as JSON"

### DISCUSSION THREAD

**Discussion by nguyendean1991** [TomorrowIdeas/plaid-sdk-php#48] [Q&A] [answered]
I used the latest plaid-sdk-php
![image](https://user-images.githubusercontent.com/80741285/140568831-f279fb37-bd91-4c63-bb23-61ecf2674d3e.png)

I tried and receive the error
![image](https://user-images.githubusercontent.com/80741285/140568889-9672ffad-255e-4adc-9307-b6e4d6f28062.png)

Please help me how to fix it. I make the params as requested but it can't work

**Comment by nguyendean1991** [comment]
I checked the object and it show valid json object
![image](https://user-images.githubusercontent.com/80741285/140592546-e7128595-390d-48ba-8f75-09b9269e19ba.png)

**Reply by brentscheffler** [reply]
That does look like valid JSON. However, the `phone_number_verified_time` format looks incorrect. According to Plaid's documentation, it should be an ISO-8601 timestamp:

> phone_number_verified_time
string
The date and time the phone number was verified in ISO 8601 format (YYYY-MM-DDThh:mm:ssZ). This field is optional, but required to enable any returning user experience.
 Only pass a verification time for a phone number that you have verified. If you have performed verification but don’t have the time, you may supply a signal value of the start of the UNIX epoch.
 Example: 2020-01-01T00:00:00Z
Format: date-time

**Reply by nguyendean1991** [reply]
Thanks  @brentscheffler
I solved by remove some addon field .
Create user by only id will make the json work
![image](https://user-images.githubusercontent.com/80741285/141144870-d574e435-21f8-4805-98c2-9cd22892a49d.png)

Problem solved

**Reply by brentscheffler** [reply]
I was able to reproduce this myself with a similar payload you had provided. I think Plaid's error message is incorrect/misleading - the JSON *is* valid, it's just the format for the `phone_number_verified_time` is incorrect.

**Comment by brentscheffler** [comment, marked as answer]
Plaid's error message is incorrect/misleading. The JSON *is* valid, it's just the payload failed to pass data type and format checks.
